Fixed: Text Color for Page Info in Issue List

qKa opened the pull request:
- Updated the text color of the page info at the bottom of the issue list to match the design specifications.

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-2)

qKa committed:
Fixed: Text Color for Page Info in Issue List

- Updated the text color of the page info at the bottom of the issue list to match the design specifications.

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-2)
